### DISCUSSION TITLE
Sort not working

### DISCUSSION THREAD

**Discussion by don-odyssey** [PHPOffice/PhpSpreadsheet#3952] [Q&A]
When I set protection for a sheet that has filters, the sort function does not work. There is no error alert, just silence

**Comment by don-odyssey** [comment]
Here is my PHP code

// set security
$protection = $spreadsheet->getActiveSheet()->getProtection();
$protection->setPassword($mslReportsPassword);
$protection->setSheet(true);
$protection->setInsertRows(true);
$protection->setFormatCells(true);
$protection->setObjects(true);
$protection->setAutoFilter(false);
$protection->setSort(true);

**Comment by oleibman** [comment]
Closing as duplicate of #3951.
